### ISSUE TITLE
Update Readme with RecetteÀMarcel syntax

### ISSUE THREAD

**Issue by brouberol** [brouberol/marcel#35]
The Readme mentions that RecetteÀMarcel files have the same syntax than dockerfiles, which is not true anymore.

**Comment by thomas-maurice** [comment]
Actually it is **half** true in the sense that a not recognized word will not be translated !

**Comment by brouberol** [comment]
True! I had not realized that the Dockerfile spec is kinda compatible with the RecetteÀMarcel one. We need to mention it.
